clang/test/AST/ast-print-sycl-kernel-call.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clang_cc1 -fsycl-is-host -ast-print %s -o - | FileCheck %s
+// RUN: %clang_cc1 -fsycl-is-device -ast-print %s -o - | FileCheck %s
+
+struct sycl_kernel_launcher {
+  template<typename KernelName, typename... Ts>
+  void sycl_kernel_launch(const char *, Ts...) {}
+
+  template<typename KernelName, typename KernelType>
+  void kernel_entry_point(KernelType kernel) {
```

Comment:
@aelovikov-intel, noticed that `[clang:sycl_kernel_entry_point(KernelName)]` is missing here.